ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global define,window*/
+
+ define([
+     'jquery',
+     'backbone'
+ ], function ($, Backbone) {
+
+  let resultTemplates = [];
+  let resultTemplateProperties = [];
+  let resultTemplatePromise = $.ajax({
+     type: 'GET',
+     context: this,
```

Comment:
Wouldn't you only need this if you used the context of this in the callback?

Reply:
yeah, nice catch, removed